CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+### Changed
+- added scripts 'check-all-gpg-keys-for-expiry.rb' and its spec script 'check_all_gpg_keys_for_expiry_spec.rb'.
```

Comment:
can you put this under an ### Added section per the [changelog docs](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md).

Reply:
This has been done.